doc/user/DESKTOP_INTEGRATION.md

```diff
@@ -21,7 +21,19 @@ sudo update-mime-database ~/.local/share/mime/
 sudo update-desktop-database ~/.local/share/applications
 ```
 
-If you have any issues, refer to the [troubleshooting](LIMITATIONS_AND_TROUBLESHOOTING.md) part.
+F3D supports rendering in environments with limited graphical contexts, including sandboxes or without Xorg running. In most cases, the default behavior will automatically detect which capabilities are available and use the right rendering backend, it is possible to specify it using `--rendering-backend` CLI option.
+
+If `libEGL.so` is available, you can use: `--rendering-backend=egl`.
+If `libOSMesa.so` is available, you can use: `--rendering-backend=osmesa`.
+
+You can use that feature for thumbnail generation in sandboxed environments (e.g., Nautilus), by modifying the `f3d.thumbnailer` file like this:
+
+```
+Exec=f3d --rendering-backend=egl --output=%o --resolution=%s,%s %i
+```
```

Comment:
So lets not focus on the `f3d.thumbnailer` file but instead focus on adding a specific line in the config file. You could write something like:


You can use that feature for thumbnail generation whenver needed (e.g., Nautilus), by modifying the [thumbnail config file](CONFIGURATION_FILE.md), eg: `~/.config/f3d/thumbnail.json` :

```
    ".*": {
         "rendering-backend": "egl"
   },
```

Reply:
Can i know where i have to add this line i.e., in desktop integration md or somewhere else